test/data/good_rules/blood_rule_twitter.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
A new section for the Twitter action should be included in https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md, including the description of the 4 parameters.